packages/ag-grid-theme/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+dist
```

Comment:
We'll want to add .DS_Store here as well, and delete the one that got pushed

Reply:
Once #3 is merged, there will be a .gitignore at the top level, and this one can be deleted entirely